src/client/app/utils/exportData.ts

```diff
@@ -15,17 +15,33 @@ import { UserRole } from '../types/items';
  */
 
 function convertToCSV(items: ExportDataSet[]) {
-	let csvOutput = 'Label,Readings,Start Timestamp\n';
-	items.forEach(set => {
-		const data = set.exportVals;
-		const label = set.label;
-		data.forEach(reading => {
-			const info = reading.y;
-			const startTimeStamp = moment(reading.x).utc().format('dddd LL LTS').replace(/,/g,''); // use regex to omit pesky commas
-			csvOutput += `"${label}",${info},${startTimeStamp}\n`; // TODO: add column for units
+	// Export line chart
+	if (items[0].exportVals[0].z === undefined) {
```

Comment:
This is fine for now. Presumably we will be adding the end date/time to all exports (2 line ones are only ones left) and then we don't need the first case.